### PR TITLE
Fix isRelayedTransactionV2

### PR DESCRIPTION
**Pull request by sstanculeanu**
- `isRelayedTransactionV2` does not check the flag anymore, only returns if the transaction is of type `RelayedTxV2` and the flag check is done on `processRelayedTxV2`. With this check on, `ComputeTransactionType` returns type `MoveBalance` for a `RelayedTxV2` with the flag disabled.

### CODE REVIEW

**Review by ccorcoveanu — APPROVED** [review]

**Review by dragos-rebegea — APPROVED** [review]

File: process/coordinator/transactionType.go [hunk 1]
```diff
@@ -183,9 +171,6 @@ func (tth *txTypeHandler) isRelayedTransactionV1(functionName string) bool {
 }
 
 func (tth *txTypeHandler) isRelayedTransactionV2(functionName string) bool {
-	if !tth.flagRelayedTxV2.IsSet() {
```
Comment (iulianpascalau) [inline process/coordinator/transactionType.go:186]: we need an import-db operation to consider this change backwards compatible

**Review by iulianpascalau — COMMENTED** [review]
not sure this PR is backwards compatible, we need an extra check

**Review by iulianpascalau — APPROVED** [review]
We merge this at is and test later in an import-db process

**Review by gabi-vuls — APPROVED** [review]
System test passed.

**Pull request merged**